### ISSUE TITLE
WASM basic setup: Version mismatch console error

### ISSUE THREAD

**Issue by GRASBOCK** [not-fl3/macroquad#569]
First off, I am glad, that all the examples I have tried so far work locally out of the box. Thanks for working on such a library!

Now I want to run it in the browser.
I set up the project as described on [https://crates.io/crates/macroquad](https://crates.io/crates/macroquad).
Compiled to WASM (issue appears in both debug and release) and used the same `index.html`. 
Running it on Firefox the canvas looks just like it should, but the console is outputting an error:
![version_mismatch](https://user-images.githubusercontent.com/31473223/221304389-ed355723-88bc-4744-8517-48ece32e17ea.png)
I downloaded the [other version](https://github.com/not-fl3/macroquad/blob/master/js/mq_js_bundle.js) (as specified in the comment) and put it next to the index.html
![version_mismatch2](https://user-images.githubusercontent.com/31473223/221307559-b108a995-50d5-4f0f-ad87-fdafc8a49f5e.png)
This bumped up the version from 0.3.11 to 0.3.13.
It won't affect the program, but I would like to know to which extend it is a problem and if it can be fixed.

Aside from the console error I also want to ask, what exactly the .js script does.

**Comment by not-fl3** [comment]
if thats the only error -  its a known problem, will fix soon!

**Comment by not-fl3** [comment]
>  However when I put the whole thing on a remote host it fails (canvas remains empty) with two error messages.

double check that remote server support application/wasm MIME type

**Comment by GRASBOCK** [comment]
Oh wow, that actually worked faster than expected. I am using nginx as a http server. So all I had to do was give it 

```
include mime.types;
types {
    application/wasm wasm;
}
```

like described [here](https://stackoverflow.com/questions/62739028/how-to-serve-the-right-mime-type-for-a-blazor-site-on-ubuntu-nginx)
Thanks so much for the quick reply :)

**Comment by emoritzx** [comment]
This error is still happening, although the `sapp-wasm` version is now `0.4.0`:
> Version mismatch: gl.js version is: 0.3.13, rust sapp-wasm crate version is: 0.4.0

**Comment by not-fl3** [comment]
> This error is still happening, although the `sapp-wasm` version is now `0.4.0`:
> 
> > Version mismatch: gl.js version is: 0.3.13, rust sapp-wasm crate version is: 0.4.0
> 
> for `macroquad = "0.4.2"`

yes, the error is still there, it should be fixed. but if thats the only error - everything should still work just fine!

**Comment by emoritzx** [comment]
can confirm :+1:  thanks!

**Comment by avelican** [comment]
> yes, the error is still there, it should be fixed. but if thats the only error - everything should still work just fine!

@not-fl3 Do you mean that this error should not prevent the game from running?

I don't get any other error, but the game works on Windows but not in WASM.

**Comment by not-fl3** [comment]
yes, if its the only error - everything should be fine!

You can check some miniquad demos, like https://not-fl3.github.io/miniquad-samples/helmet.html (takes some time to load) to see if its a browser/OS problem and some sort of a macroquad bug or the problem is in the build setup.

**Comment by not-fl3** [comment]
> > yes, the error is still there, it should be fixed. but if thats the only error - everything should still work just fine!
> 
> @not-fl3 Do you mean that this error should not prevent the game from running?
> 
> I don't get any other error, but the game works on Windows but not in WASM.

yes, if its the only error - everything should be fine!

You can check some miniquad demos, like https://not-fl3.github.io/miniquad-samples/helmet.html (takes some time to load) to see if its a browser/OS problem and some sort of a macroquad bug or the problem is in the build setup.

**Comment by avelican** [comment]
The other examples run fine!
Might be an issue with my game, this is basically my first Rust program. 
It does run on Windows, though there are some graphical glitches (the original game used a different rendering system, I need to rewrite some of the math to account for the differences.)
https://andai.tv/games/pong-goof/
https://github.com/avelican/pong-rs

**Comment by not-fl3** [comment]
I would recommend trying to build smaller examples to make sure that your build setup is correct, to have a point of reference with something that works
